app/serializers/user_export_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class UserExportSerializer < ApplicationSerializer
+  attributes :id, :filename, :uri, :filesize, :extension, :retain_hours, :human_filesize
+
+  def filename
+    object.upload.original_filename
+  end
+
+  def uri
+    object.upload.short_path
+  end
+
+  def filesize
+    object.upload.filesize
+  end
+
+  def extension
+    object.upload.extension
+  end
+
+  def retain_hours
```

Comment:
I _think_ for 1:1 attributes on the object you don't need to define a method in the serializer